### PR TITLE
Fixes for project for iaac branch

### PR DESCRIPTION
**Pull request by cooktheryan**
Signed-off-by: Ryan Cook <rcook@redhat.com>

### CODE REVIEW

File: config/default/kustomization.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: awx-resource-operator-system
+namespace: ansible-automation-platform
```
Comment (rooftopcellist) [inline config/default/kustomization.yaml]: For upstream, can we make this awx, or awx-resource?

I can add logic midstream to convert this to `aap` for downstream (starting in 2.3, the default namespace is aap)

File: config/manager/kustomization.yaml [hunk 2]
```diff
@@ -10,3 +10,7 @@ configMapGenerator:
   name: awx-resource-manager-config
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+images:
```
Comment (rooftopcellist) [inline config/manager/kustomization.yaml]: We probably don't want to commit this kustomization change.  Could .gitignore it, as `IMG=yours make deploy` will keep generating it.

Comment (cooktheryan) [reply config/manager/kustomization.yaml]: resolved

File: roles/job/tasks/main.yml [hunk 3]
```diff
@@ -99,6 +99,10 @@
   set_fact:
     _runner_image: "{{ _custom_runner_image | default(lookup('env', 'RELATED_IMAGE_ANSIBLE_JOB_RUNNER_IMAGE')) }}"
 
+- name: Set Runner Pull Policy
+  set_fact:
+    _runner_pull_policy: "{{ runner_pull_policy | default('Always') }}"
```
Comment (rooftopcellist) [inline roles/job/tasks/main.yml]: Can we make this `IfNotPresent`?

File: roles/workflow/tasks/main.yml [hunk 4]
```diff
@@ -99,6 +99,10 @@
   set_fact:
     _runner_image: "{{ _custom_runner_image | default(lookup('env', 'RELATED_IMAGE_ANSIBLE_JOB_RUNNER_IMAGE')) }}"
 
+- name: Set Runner Pull Policy
+  set_fact:
+    _runner_pull_policy: "{{ runner_pull_policy | default('Always') }}"
```
Comment (rooftopcellist) [inline roles/workflow/tasks/main.yml]: IfNotPresent as the default

**Review by rooftopcellist — APPROVED** [review]
Thank you for these changes! I tested it out locally and it works as expected.

**Pull request merged**